db/models.py

```diff
@@ -1 +1,16 @@
+from symtable import Class
```

Comment:
The import from `symtable` is unnecessary as it is not used in this file. Consider removing it to clean up the code.